Libraries/Geolocation/Geolocation.js

```diff
@@ -216,6 +220,28 @@ var Geolocation = {
       }
       subscriptions = [];
     }
+  },
+
+  addEventListener: function(eventName: LocationEventName, callback: Function) {
```

Comment:
@grabbou Hey! Do you have any concerns related to event-subscribing API here?

Reply:
I indeed saw a comment of @grabbou on a different PR where he did put some remarks on the event-subscribing, but I figured that since on other places the `addEventListener` conversion is still being used I should follow that.

To make it compatible with other implementation throughout the source code I should probably also `return { remove: () => null }` for the warning case.

Reply:
Honestly, I am okay with any approach, I just want to have single one pattern across whole codebase. I also don't want to have it overengineered.